### PR TITLE
Register and make model Profile

### PR DESCRIPTION
**Pull request by dangthihoa**
Related ticket: https://edu-redmine.sun-asterisk.vn/issues/27337
what:  make model Profile and Can Register
Screenshot from 2020-08-21 15-53-06

### CODE REVIEW

**Review by Atanna128 — APPROVED** [review]

**Review by HieuNT-Sun-Asterisk — APPROVED** [review]

**Comment by dangthihoa** [comment]
ready

File: travel/templates/register.html [hunk 1]
```diff
@@ -14,44 +14,61 @@
...
+                                    {% endfor %}
+                                    {% for error in form.non_field_errors %}
+                                        <div class="form-alert text-center mb-2 w-100">
+                                            <strong>{{ error|escape }}</strong>
+                                        </div>
+                                    {% endfor %}
+                                {% endif %}
 
 
                                 <div class="form-group">
```
Comment (thaodtd-0665) [inline travel/templates/register.html:33]: dùng luôn form đi em, đừng dùng form html thuần nữa

Comment (dangthihoa) [reply travel/templates/register.html:33]: dung form xau lam chi aj

**Comment by dangthihoa** [comment]
ready

**Review by thaodtd-0665 — APPROVED** [review]

**Pull request merged**